learn/index.html

```diff
@@ -154,6 +154,14 @@
 
       <ul class="cards">
         <li>
+          <h3>Lunch &amp; Learn</h3>
```

Comment:
I wasn't sure what to put here. My presentation was not at a conference. It was over lunch at my workplace ([OrgSync](http://www.orgsync.com)).